packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx

```diff
@@ -152,9 +152,9 @@ export function ButtonMenu({ info, onUpdate, onClose, onRemove, value, withNewTa
                 Button settings
             </Toolbox.Header>
 
-            {info && info.length && (
+            {Boolean(info && info.length) && (
                 <Toolbox.Section>
-                    <InfoText className={styles.info}>{info.map(renderInfoText)}</InfoText>
+                    <InfoText className={styles.info}>{info?.map(renderInfoText)}</InfoText>
```

Comment:
Just to share. 

A slightly more elegant solution could be to have a default value for `info` during destructuring:

```ts
export function ButtonMenu({ info = [], ... }: Props) {
```

and then only check for `.length`:

```tsx
            {info.length > 0 && (
                <Toolbox.Section>
                    <InfoText className={styles.info}>{info?.map(renderInfoText)}</InfoText>
```